src/training/day4/Homework4.java

```diff
@@ -0,0 +1,19 @@
+package training.day4;
+
+public class Homework4 {
+    public static void main(String[] args){
+
+        Vehicle AudiA4 = new Vehicle(20, 150 );
```

Comment:
Do not keep empty space after the last argument in object if no other arguments are expected. It is for all the other objects in the class.